test/test-decl.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ *
+ * Compile or run this file to see if type checks
+ *
+ * required package (not included in package.json):
+ * - ts-node or tsc
+ * - @types/chai
```

Comment:
Feel free to add these as dev dependencies, and to include these tests in the `npm test` machinery.